app/services/analytics_events.rb

```diff
@@ -6289,7 +6289,7 @@ def saml_auth(
     finish_profile:,
     requested_ial:,
     request_signed:,
-    matching_cert_serial:,
+    matching_cert_serial: nil,
```

Comment:
Is the default needed if it's still being assigned in the method call?

Reply:
seems like it works! maybe i had hit a transient error or something